packages/api-specification/transform-type-info.js

```diff
@@ -150,7 +150,7 @@ const documentClass = (className) => {
           h5('Returns'),
           dl([
             dt(formatType(d.match.type), {class: 'code return-value'}),
-            dd(formatComment(d.match.comment))
+            dd(d.match.comment ? d.match.comment.returns : '')
```

Comment:
Yeah that worked, but for some methods it would print both the method description and the return text, so the method text was still being repeated. e.g.

```
"comment": {
  "shortText": "Flashes the window's frame and taskbar icon.",
  "returns": "A promise which resolves to nothing when the function has completed.\n"
}
```
Displays both text elements under the return type like so:

---

### flashFrame
Flashes the window's frame and taskbar icon.

**Returns:** `Promise<void>`
Flashes the window's frame and taskbar icon.
A promise which resolves to nothing when the function has completed.

---
Which looked a bit odd to me.